### PR TITLE
change orchestrator query to cluster

### PR DESCRIPTION
**Pull request by sunkickr**
## Description

Updates the GetOrchestrator query to GetCLuster because GetOrchestrator is no longer maintained in the Astro API

## 🎟 Issue(s)

- https://github.com/astronomer/astro-cli/issues/588

## 🧪 Functional Testing

All tests updated and pass

## 📋 Checklist

- [X] Rebased from the main (or release if patching) branch (before testing)
- [X] Ran `make test` before taking out of draft
- [X] Ran `make lint` before taking out of draft
- [X] Added/updated applicable tests
- [X] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: astro-client/queries.go [hunk 1]
```diff
@@ -107,12 +107,21 @@ var (
 	}
 	`
 
-	GetOrchestrators = `
-	query orchestrators($input: OrchestratorInput!) {
-		orchestrators(input: $input) {
+	// GetOrchestrators = `
+	// query orchestrators($input: OrchestratorInput!) {
+	// 	orchestrators(input: $input) {
+	// 		id
```
Comment (kushalmalani) [inline astro-client/queries.go:113]: Can we just clean this up, the comments

**Review by kushalmalani — APPROVED** [review]
Can we add a screenshot of the local build working against the dev env?

File: astro-client/queries.go [hunk 2]
```diff
@@ -107,12 +107,21 @@ var (
...
+	// 		cloudProvider
+	// 		name
+	// 	}
+	// }
+	// `
```
Comment (neel-astro) [inline astro-client/queries.go:118]:
```suggestion
```

**Review by neel-astro — APPROVED** [review]
looks fine, apart from one minor comment

**Comment by sunkickr** [comment]
I added an issue and removed the commented out code

**Pull request merged**